vcpkg/contributing/maintainer-guide.md

```diff
@@ -115,17 +115,27 @@ Examples:
 
 ### Install copyright file
 
-Each port has to provide a file named `copyright` in the folder `${CURRENT_PACKAGES_DIR}/share/${PORT}`.
+Each port has to provide a file named `copyright` in the folder `${CURRENT_PACKAGES_DIR}/share/${PORT}`. If the license content for a port is available within the sources, this file should be created by a call to [`vcpkg_install_copyright()`](../maintainers/functions/vcpkg_install_copyright.md). `vcpkg_install_copyright` also bundles multiple copyright files if necessary.

```

Comment:
```suggestion
Each port has to provide a file named `copyright` in the folder `${CURRENT_PACKAGES_DIR}/share/${PORT}`. If a port's license content is available within its source files, this file should be created by a call to [`vcpkg_install_copyright()`](../maintainers/functions/vcpkg_install_copyright.md). `vcpkg_install_copyright` also bundles multiple copyright files if necessary.

```

Reply:
I intentionally did not use this construction because I was worried that describing something as "the port's license" implied the license that applied to the port itself, that is, the directory where `vcpkg.json` and `portfile.cmake` live. We really want some way to say 'the license associated with the content that the port installs'. I think 'the content that the port installs' has been associated with the word 'package' but merging artifacts really made that even more confusing...

What are your thoughts?

Reply:
I think it's reasonable to change the word "port" to "package" in this context. The package is what ultimately ends up installed on the machine so that way there should be less confusion.